packages/flutter/lib/src/foundation/change_notifier.dart

```diff
@@ -178,4 +178,7 @@ class ValueNotifier<T> extends ChangeNotifier {
     _value = newValue;
     notifyListeners();
   }
+
+  @override
+  String toString() => 'ValueNotifier(value: $value)';
```

Comment:
replace `ValueNotifier` with `$runtimeType` so that if someone subclasses it it's still right